### DISCUSSION TITLE
Create changelog website for rich

### DISCUSSION THREAD

**Discussion by JonasHiltl** [Textualize/rich#3556] [Show and tell]
Hi, I'm the creator of [Openchangelog](https://github.com/JonasHiltl/openchangelog) and a big fan of rich.

I created a changelog website for rich using Openchangelog which you can check out at [rich.openchangelog.com](https://rich.openchangelog.com/).
It automatically stays in sync with your `CHANGELOG.md` file.

Would love to have this linked in your `CHANGELOG.md` or docs to make it more accessible. After all who really wants to read a `CHANGELOG.md` file when they could have a beautiful, styled website.

I also created this badge which is useful for README's: [![Changelog Badge](https://img.shields.io/badge/changelog-black)](https://rich.openchangelog.com)

**Comment by edward-jazzhands** [comment]
Hey there, just wanted to say this looked interesting but I tried to view the URL and it seems to be broken now. I'm getting this error

> Get "https://api.github.com/repos/textualize/rich/contents/CHANGELOG.md": could not refresh installation id 54120967's token: received non 2xx response status "404 Not Found" when fetching https://api.github.com/app/installations/54120967/access_tokens
